### ISSUE TITLE
Subject column selection step does not reflect KB settings

### ISSUE THREAD

**Issue by brodecva** [odalic/odalic-ui#169]
_From @tomas-knap on November 18, 2016 15:30_

I set up that I would like to use DBpedia and GermanDbpedia. But in the subject column detection, I can see also DBPedia Clone!

![screen shot 2016-11-18 at 16 29 17](https://cloud.githubusercontent.com/assets/3014917/20435380/2c801114-adac-11e6-9295-3131afd759c8.png)

Solution: This step should show only relations which were defined in the config file!

_Copied from original issue: odalic/sti#219_

**Comment by brodecva** [comment]
> Solution: This step should show only relations which were defined in the config file!

As we postponed any KB management, it is currently accepted that the available KBs on the server and client must be manually aligned.

**Comment by tomas-knap** [comment]
The depicted knowledge bases (for which subject column may be selected) should reflect the knowledge bases defined for the task during task configuration. 

Depends on #12.

**Comment by brodecva** [comment]
Duplicates #257 .
